encord/orm/model.py

```diff
@@ -39,6 +45,41 @@ class Model(base_orm.BaseORM):
     DB_FIELDS = OrderedDict([("model_operation", int), ("model_parameters", dict)])
 
 
+@dataclass
+class ModelConfiguration(Formatter):
+
+    model_uid: str
+    title: str
+    description: str
+    # The corresponding feature node hashes of the ontology object
+    feature_node_hashes: List[str]
+    model: AutomationModels
+
+    @classmethod
+    def from_dict(cls, json_dict: dict):
+        return ModelConfiguration(
+            model_uid=json_dict["model_hash"],
+            title=json_dict["title"],
+            description=json_dict["description"],
+            feature_node_hashes=cls._get_feature_node_hashes(json_dict["features"]),
+            model=cls._get_automation_model(json_dict["model"]),
+        )
+
+    @classmethod
+    def _get_feature_node_hashes(cls, features: dict) -> List[str]:
+        return list(features.keys())
+
+    @classmethod
+    def _get_automation_model(cls, automation_model_str: str) -> AutomationModels:
+        try:
+            return AutomationModels(automation_model_str)
+        except ValueError as e:
```

Comment:
@frederik-encord I guess in hindsight this can be a bit annoying for clients with the enum.

Reply:
Not sure why this would be an issue? The main idea of the "endpoint" is to allow people to find the model they want to run inference with or train. So the enum just allows filtering on the type of model you are looking for.

What would be the annoying part @denis-cord?

Reply:
I'm just thinking of the following scenario:
1) client is at SDK version X
2) we add a new model, which means that here in version X we would throw 
3) client is forced to update their SDK version

Not a big issue for now that we only do minor version updates, but definitely annoying once we introduce a breaking change and force the client to deal with it.

Reply:
Yea that is not great. @denis-cord is there a way to get both intellisense and avoid the need to update sdk for new models in the future?